events/events.go

```diff
@@ -361,6 +366,41 @@ type OperationDeprovisionedData struct {
 	Team   *Team        `json:"team,omitempty"`
 }
 
+// OperationResized represents a resize operation event.
+type OperationResized struct {
+	BaseBody
+	Data OperationResizedData `json:"data"`
+}
+
+// OperationResizedData holds the event specific data.
+type OperationResizedData struct {
+	OperationID manifold.ID `json:"operation_id"`
+
+	ResourceID manifold.ID `json:"resource_id"`
+	Resource   *Resource   `json:"resource,omitempty"`
+
+	UserID *manifold.ID `json:"user_id,omitempty"`
```

Comment:
Hmm, I was just going to ask why are these separate because we could just have an `Owner` because the IDs are typed, but there User and Team structures are not compatible; you could have an `OwnerID` and then have `OwnerUser` and `OwnerTeam` I don't know if that's any nicer though; I guess the one benefit is that you could make `OwnerId` required, rather than two optional fields.
If you like the idea but not the name, `Actor` could be an alternative.

Reply:
I think it's the pattern established with the other events.

Reply:
Yeah. That follows the other specs we have. For example a operation and resource have both fields. Not really my preference, but, following the same pattern.

Reply:
:+1: